gpsrenda/widgets/bargauges.py

```diff
@@ -11,39 +11,37 @@ def __init__(self, x, y, w = 600, h = 60, label = '{val:.0f}', dummy_label = '99
         self.h = h
         self.label = label
         self.caption = caption
-        
+
         self.padding = h / 8
 
         if dummy_caption == None:
             dummy_caption = caption
         self.caption_text = Text(self.x + self.w - self.padding / 2,
                                  self.y + self.h - self.padding,
                                  size = self.h * 0.5,
-                                 dropshadow = self.h * 0.1,
+                                 dropshadow = 0,
```

Comment:
not a fan?  maybe have these configurable in `globals`?

Reply:
I noticed the way you did dropshadow results in the text being bumped up, so it appears misaligned with hbars, for example.

The current thinking on UI is to keep things simple - minimize gradients and text effects.